### PR TITLE
Orchestration now uses the HTTP API instead of mongo

### PR DESCRIPTION
**Pull request by jawher**
In order to do things like nark jobs and variants as finished, add SCM info, ...

### CODE REVIEW

**Comment by haklop** [comment]
LGTM!

File: cli/bzk/service.go [hunk 1]
```diff
@@ -460,10 +464,11 @@ func startContainer(client *docker.Docker, options *docker.RunOptions, allContai
 
 }
 
-func getServerEnv(home, dockerSock, scmKey, syslogURI, mongoURI string) map[string]string {
+func getServerEnv(home, dockerSock, scmKey, apiURI, syslogURI, mongoURI string) map[string]string {
 	envMap := map[string]string{
 		"BZK_HOME":       home,
 		"BZK_DOCKERSOCK": dockerSock,
+		"BZK_API_URL":    apiURI,
```
Comment (julienvey) [inline cli/bzk/service.go]: Any reason it's named "URL" here instead of "URI" ? and in context.go also 
We should try to be consistent with the naming and have URI everywhere in the code

Seems the same problem with SYSLOG_URL/I

Comment (jawher) [inline cli/bzk/service.go]: We'll do that in a separate PR
